sqlite/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/ardanlabs/python-go/trades
+module github.com/ardanlabs/python-go/sqlite
```

Comment:
I think it's confusing that the import path ends with `sqlite` but the package name is `trades`.
We should probably have all go related code mode to the `trades` directory.